src/Illuminate/Support/Str.php

```diff
@@ -30,6 +30,24 @@ class Str
     protected static $studlyCache = [];
 
     /**
+     * Return the remainder of a string after a given value.
+     *
+     * @param  string  $subject
+     * @param  string  $search
+     * @return string
+     */
+    public static function after($subject, $search)
+    {
+        if (! static::contains($subject, $search)) {
+            return $subject;
+        }
+
+        $searchEndPos = strpos($subject, $search) + static::length($search);
```

Comment:
`strpos` is single-byte while the rest of the code is utf8 aware.
Therefore you get wrong results with `after('é foobarbaz', 'foo')`